api/test/pact/pact-tests/wa2/getSearchResults.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect } from 'chai';
+import * as config from 'config';
+import * as sinon from 'sinon';
+import { mockReq, mockRes } from 'sinon-express-mock';
+import { PactTestSetup } from '../settings/provider.mock';
+import { getCcdDataAPIOverrides } from '../utils/configOverride';
+import { requireReloaded } from '../utils/moduleUtil';
+
+const { Matchers } = require('@pact-foundation/pact');
+const { somethingLike } = Matchers;
+
+const pactSetUp = new PactTestSetup({ provider: 'ccdDataStoreAPI_search', port: 8000 });
```

Comment:
Not sure if broker address is right, consistent with our understanding via confluence page but not consistent with https://pact-broker.platform.hmcts.net/?search=ccdData. Maybe ask Mo or Sreekanth about it? Let me know if they have an answer.

Reply:
Sreekanth  said "This is fine if test is passing locally. The broker name mapping will be in another file and will be mapped if there is some issue during deployment"